index.html

```diff
@@ -61,16 +64,35 @@
     -->
     <link defer rel="import" href="../px-dark-theme/px-dark-theme-styles.html">
     <link defer rel="import" href="../px-dark-demo-theme/px-dark-demo-theme-styles.html">
-    <link defer rel="import" href="../px-demo/px-demo-theme-switcher.html">
+    <!-- <link defer rel="import" href="../px-demo/px-demo-theme-switcher.html">
     <px-demo-theme-switcher-frame id="themeMessenger" origin="https://www.predix-ui.com"></px-demo-theme-switcher-frame>
-    <px-demo-theme-switcher id="themeSwitcher"></px-demo-theme-switcher>
+    <px-demo-theme-switcher id="themeSwitcher"></px-demo-theme-switcher> -->
+
+    <label for="theme-switcher" class="theme-switcher-checkbox">
+      <input type="checkbox" id="theme-switcher">
+      Dark theme
+    </label>
+
     <script>
       (function() {
-        var messenger = document.getElementById('themeMessenger');
-        var switcher = document.getElementById('themeSwitcher');
-        messenger.addEventListener('is-dark-theme-changed', function(evt) {
-          switcher.isDarkTheme = evt.detail.value;
-        });
+        // var messenger = document.getElementById('themeMessenger');
```

Comment:
Is comment needed?

Reply:
Yeah, I left those intentionally. Probably some additional steps will be required here in order to integrate with https://predix-ui.com and this will be needed